romanesco/plugins/spark/pyspark_executor.py

```diff
@@ -6,7 +6,7 @@
 def run(task, inputs, outputs, task_inputs, task_outputs, **kwargs):
     from . import SC_KEY
 
-    custom = imp.new_module("custom")
+    custom = imp.new_module("__romanesco__")
```

Comment:
Maybe?  they would have to be dynamically changing the ```mode``` in the task metadata but using the same script file. I could see reasons why you would want to do this,  but it would be just as easy to condition on the ```sc``` variable under the ```___romanesco___``` entry point.   My vote is for consistent entry points across python and spark.python modes.